src/components/twilio-voice-basic-call-control/routes.js

```diff
@@ -7,7 +7,7 @@ const { accountSid, apiKeySid, apiKeySecret } = config;
 const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
 
 router.post('/conferences/:conferenceSid/participants/:callSid', async (req, res) => {
-  const { hold } = req.body;
+  const { hold } = JSON.parse(req.body);
```

Comment:
I think if you set the headers to `application/json` in the fetch, you don't need to do `JSON.parse` here.